docs/resources/vdb.md

```diff
@@ -178,6 +187,8 @@ Environment variable to be set when the engine creates a VDB. See the Engine doc
 
 * `snapshot_id` - (Optional) The ID of the snapshot from which to execute the operation. If the snapshot_id is not, selects the latest snapshot.
 
+* `bookmark_id` - (Optional) The ID of the bookmark from which to execute the operation. The boomkark must contain only one VDB.
```

Comment:
boomkark -> bookmark

Reply:
Here and elsewhere, we're writing the "the ID of XXX", but maybe we should say "The ID or name of XXX"

Reply:
done